app/src/main/res/xml/method.xml

```diff
@@ -296,6 +297,15 @@
             android:imeSubtypeExtraValue="KeyboardLayoutSet=MAIN:qwertz,AsciiCapable,SupportTouchPositionCorrection,EmojiCapable"
             android:isAsciiCapable="true"
     />
+        <subtype android:icon="@drawable/ic_ime_switcher"
```

Comment:
Please adjust the indent to be like other subtypes

Reply:
This line has 8 spaces indent, while (almost) all others have 4 spaces indent in the `<subtype` line.